.github/workflows/docker_image.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Docker meta
         id: docker_meta
-        uses: crazy-max/ghaction-docker-meta@dd2e615
+        uses: crazy-max/ghaction-docker-meta@dd2e615c98e39396339772bcbedf2ab7fd12d512
```

Comment:
Can we use a tag name instead of a commit? This one seems to refer to https://github.com/crazy-max/ghaction-docker-meta/releases/tag/v1.7.0